RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    internal class CItem:IComparable<CItem>
+    {
+
+        private static int countObj = 0;
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+            //set { id = countObj; }
+
+        }
+        private string name;
+
+        public string Name
+        {
+            get { return name; }
+            set
+            {
+                if (!value.Equals(""))
+                    name = value;
+                else
+                    throw new ArithmeticException("Illegal name, name must be string");
+            }
+        }
+
+        private int idShelf;
+
+        public int IdShelf
+        {
+            get { return idShelf; }
+            set
+            {
+                if (value >= 0)
+                    idShelf = value;
+                else
+                    throw new ArithmeticException("Illegal idShelf, idShelf must be int");
+            }
+        }
+
+        private string typeI;
+
+        public string TypeI
+        {
+            get { return typeI; }
+            set
+            {
+                if (value.Equals("food") || value.Equals("Food") || value.Equals("Drink") || value.Equals("drink"))
```

Comment:
When you have a closed set of options that are fixed you should use Enums, this is true also for Kashroot.
This can help you in the problem of checking upper and lower cases too

Reply:
fixed